xpi-manifest.yml

```diff
@@ -35,6 +38,16 @@ xpis:
       - web-ext-artifacts/normandy-devtools.zip
     addon-type: privileged
     install-type: yarn
+  - name: normandy-devtools-1.x
+    description: Normandy Devtools
+    repo-prefix: normandydevtools
+    active: true
+    private-repo: false
+    branch: 1.x
```

Comment:
1.x-minimal is the branch we're actually working off right now

Reply:
Fixed, and added to the protected branch list.